custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -292,18 +297,27 @@ def _followups(self):
 
     @property
     def state(self):
+        if self.test_phi is not None:
+            return MockLocation('FAKESTATE', 'fake_state_id', MockLocationType('state', 'state'))
```

Comment:
Good idea. @nickpell do you think this is still useful for testing? otherwise I'll leave it as is.